library/patient.inc

```diff
@@ -812,6 +814,15 @@ function getPatientPID($args)
 
     return $returnval;
 }
+function getPatientDoBonly($pid)
```

Comment:
Since this is a new function, I'd say make the name `getPatientDateOfBirth` - the DoB camel case is kinda weird and the `only` part isn't really necessary.